### ISSUE TITLE
Map not working in 4.1.6

### ISSUE THREAD

**Issue by Admonstrator** [stoo2000/membermap#7]
Hi there,

I'm using the Member Map (3.0.0 Beta 1) on my community and everything was working well. Yesterday I upgraded to version 4.1.6 and the map isn't working anymore.
Instead I'm receiving many errors like EX0 if I want to visit the Map page. One weird thing is that the user profiles don't work anymore if the Member Map app is active. Even the hover cards fail to load.
If I try to visit a profile of one of my members this error will appear:

```
Mon, 04 Jan 2016 17:11:43 +0000 (Severity: 2)
https://www.mycommunity.de/profil/29-admonstrator/
UnexpectedValueException
0: 
#0 /www/htdocs/mycommunity.de/applications/core/modules/front/members/profile.php(263): IPS\_Theme->getTemplate('profile')
#1 /www/htdocs/mycommunity.de/system/Dispatcher/Controller.php(94): IPS\core\modules\front\members\_profile->manage()
#2 /www/htdocs/mycommunity.de/applications/core/modules/front/members/profile.php(65): IPS\Dispatcher\_Controller->execute()
#3 /www/htdocs/mycommunity.de/system/Dispatcher/Dispatcher.php(129): IPS\core\modules\front\members\_profile->execute()
#4 /www/htdocs/mycommunity.de/index.php(13): IPS\_Dispatcher->run()
#5 {main}
```

In version 4.1.5.x everything worked well.

**Comment by MartinAronsen** [comment]
@stoo2000 You have any idea what changed in 4.1.6 to cause this to happen?
I don't have time right now to investigate and release a hotfix, so it'll have to wait till next week unless it's a quick one you can do.

**Comment by MartinAronsen** [comment]
Fixed in e7e9aa3.
